core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -436,12 +436,24 @@ oppia.directive('conversationSkin', [
             }
 
             if (ExplorationPlayerStateService.isStateTerminal(stateName)) {
-              ExplorationRecommendationsService.getRecommendedSummaryDicts(
-                ExplorationPlayerStateService.getAuthorRecommendedExpIds(
-                  stateName),
-                function(summaries) {
-                  $scope.recommendedExplorationSummaries = summaries;
-                });
+              $scope.parentExplorationIds = UrlService.
```

Comment:
I know this is a bit different to what I suggested earlier (sorry), but in the spirit of keeping URL stuff within UrlService and non-URL stuff outside it, can we do UrlService.getParamValuesAsList('parent') or similar here?

Reply:
Yes, OK, with $location.search removed, this should be possible.

Reply:
Done